aws/resource_aws_ssm_patch_baseline.go

```diff
@@ -124,13 +155,25 @@ func resourceAwsSsmPatchBaseline() *schema.Resource {
 				Default:      ssm.PatchComplianceLevelUnspecified,
 				ValidateFunc: validation.StringInSlice(ssm.PatchComplianceLevel_Values(), false),
 			},
+			"rejected_patches_action": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringInSlice(ssm.PatchAction_Values(), false),
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+					return old == ssm.PatchActionAllowAsDependency && new == ""
+				},
+			},
```

Comment:
`DiffSuppressFunc` may go away in the future -- `Computed` should be used to signal to Terraform that the value is unknown if not configured or use a `Default`. I don't have time to verify the correct behavior right now with this resource, so I will change this to `Computed` as it is safer.

```suggestion
			"rejected_patches_action": {
				Type:         schema.TypeString,
				Optional:     true,
				Computed:     true,
				ValidateFunc: validation.StringInSlice(ssm.PatchAction_Values(), false),
			},
```